### DISCUSSION TITLE
not supported in the v1/chat/completions endpoint

### DISCUSSION THREAD

**Discussion by Isomorph70** [danielmiessler/Fabric#1121] [Q&A] [answered]
Is this an error? Or am I doing it wrong? 

`$ cat test_case_1_out.txt | fabric -m "davinci-002" -p summarize

error, status code: 404, message: This is not a chat model and thus not supported in the v1/chat/completions endpoint. Did you mean to use v1/completions?
`

**Comment by SimonMagusPY** [comment, marked as answer]
Hi,

Rather use "gpt-4o-mini" . Da vinci models not supported anymore. Mini also has the larger context window, perfect for fabric.
